rtloader/test/python/datadog_checks/base/checks/__init__.py

```diff
@@ -1,5 +1,5 @@
 # AgentCheck stubs for testing
-class AgentCheck(object):
+class AgentCheck(object):  # noqa
```

Comment:
What did anger the linter here? Would it make sense to skip specific lint only?

Reply:
This one https://docs.astral.sh/ruff/rules/useless-object-inheritance/. Since this is probably used on py2, I did not want to update that